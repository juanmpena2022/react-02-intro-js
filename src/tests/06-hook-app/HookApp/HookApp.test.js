import { shallow } from "enzyme";
import { HookApp } from "../../../06-hook-app/components/HookApp/HookApp";

describe("pruebas del componente <HookApp />", () => {
  const wrapper = shallow(<HookApp />);

  test("snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
