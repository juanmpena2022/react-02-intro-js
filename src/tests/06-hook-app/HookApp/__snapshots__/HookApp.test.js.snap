// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas del componente <HookApp /> snapshot 1`] = `
<div>
  Hola Mundo!!!
</div>
`;
